### ISSUE TITLE
No mention of installing Microsoft.WindowsAzure.ConfigurationManager package for Azure Storage in doc

### ISSUE THREAD

**Issue by garrardkitchen** [dotnet/docs#1333]
Hi,

The documentation for all F# Storage (e.g. https://docs.microsoft.com/en-us/dotnet/articles/fsharp/using-fsharp-on-azure/table-storage) does not include anything regarding the installing of the Microsoft.WindowsAzure.ConfigurationManager package. This must be installed for CloudConfigurationManager to be available.  This is true for VSCode and f# console app and VS f# console app.

```
let storageConnString = 
    CloudConfigurationManager.GetSetting("StorageConnectionString")
```

Warmest regards,

Garrard.

**Comment by mairaw** [comment]
@cartermp can you take a look at this one?

**Comment by cartermp** [comment]
Thanks for the report, @garrardkitchen.  I'll take a look.

**Comment by mairaw** [comment]
This issue seems to be fixed with PR #2834. @cartermp can you confirm?

**Comment by cartermp** [comment]
Yep!
